config.py

```diff
@@ -5,12 +5,16 @@
 
 env = "prod"
 if env == "local":
-    print("Using release configuration...")
+    print("Using local configuration...")
     SQLALCHEMY_DATABASE_URI = os.environ.get("SQLALCHEMY_DATABASE_URI_OPEN")
     SQLALCHEMY_TRACK_MODIFICATIONS = False
-    BASE_URL = 'http://0.0.0.0:5000/'
-elif env == 'prod':
-    print('Using release configuration...')
-    SQLALCHEMY_DATABASE_URI = os.environ.get('SQLALCHEMY_DATABASE_URI_OPEN')
+    BASE_URL = "http://0.0.0.0:5000/"
+    DEBUG = True
+    SECRET_KEY = "my-secret-key"
+elif env == "prod":
+    print("Using release configuration...")
+    SQLALCHEMY_DATABASE_URI = os.environ.get("SQLALCHEMY_DATABASE_URI_OPEN")
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     BASE_URL = "https://openqairamapnapi.qairadrones.com/"
+    DEBUG = False
+    SECRET_KEY = "my-secret-key"  # TODO: Replace with a more secure key, and don't push it to Github
```

Comment:
This is my first question, @GersonMontesinos. This secret key shouldn't be written here even if it's open source, for security reasons. One alternative would be to do this:

```suggestion
    SECRET_KEY = os.environ.get("SECRET_KEY")
```

But then in your production environment you would need to set this environment variable. How would you like to proceed?